src/newcharacter.cpp

```diff
@@ -2293,11 +2296,19 @@ tab_direction set_description(WINDOW *w, player *u, const bool allow_reroll, poi
         const std::string action = ctxt.handle_input();
 
         if (action == "NEXT_TAB") {
-            if( OPTIONS["POINT_DISTRIBUTION"] != "freeform" && !points.is_valid() ) {
-                popup(_("Too many points allocated, change some features and try again."));
+            if (OPTIONS["POINT_DISTRIBUTION"] != "freeform" && !points.is_valid() ) {
+                if (points.skill_points_left() < 0 ) {
+                        popup(_("Too many points allocated, change some features and try again."));
```

Comment:
It's better to replace general "points" by "skill points" here, just to be in line with other popups.